test/jdk/javax/swing/border/Test4129681.java

```diff
@@ -21,39 +21,52 @@
  * questions.
  */
 
-/*
- * @test
- * @bug 4129681
- * @summary Tests enabling/disabling of titled border's caption
- * @author Sergey Malenkov
- * @run applet/manual=yesno Test4129681.html
- */
-
-import java.awt.BorderLayout;
 import java.awt.event.ItemEvent;
-import java.awt.event.ItemListener;
 import javax.swing.BorderFactory;
-import javax.swing.JApplet;
+import javax.swing.Box;
 import javax.swing.JCheckBox;
+import javax.swing.JComponent;
 import javax.swing.JLabel;
 
-public class Test4129681 extends JApplet implements ItemListener {
-    private JLabel label;
-
-    @Override
-    public void init() {
-        JCheckBox check = new JCheckBox("disable");
-        check.addItemListener(this);
+/*
+ * @test
+ * @bug 4129681
+ * @summary Tests enabling/disabling of titled border's caption
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual Test4129681
+ */
 
-        this.label = new JLabel("message");
-        this.label.setBorder(BorderFactory.createTitledBorder("label"));
-        this.label.setEnabled(!check.isSelected());
+public class Test4129681 {
+    public static void main(String[] args) throws Exception {
+        String testInstructions = """
+                Turn on the checkbox to disable the label.
```

Comment:
Can I suggest something like this ?
```suggestion
                Click the checkbox to disable/enable the label.
```

Reply:
Since the test is for disable, I would retain the instructions to disable.

Reply:
[Microsoft recommends](https://learn.microsoft.com/en-us/windows/win32/uxguide/ctrl-check-boxes#documentation) using *select / clear* the check box:

<q cite="https://learn.microsoft.com/en-us/windows/win32/uxguide/ctrl-check-boxes#documentation">To describe user interaction, use select and clear.</q>

I don't know about Apple macOS and Linux systems.